prime-router/metadata/fhir_mapping/hl7/extension/ExtensionHierarchicDesignator.yml

```diff
@@ -0,0 +1,55 @@
+url:
```

Comment:
Used to implement [this mapping](https://docs.google.com/spreadsheets/d/1thMC8Et_hA-VC-DyerLrpNDcNdzPOIql9Z7SP7u9h_w/edit#gid=0) in a way general enough to be used as a generic custom extension for HD mappings in fields other than `assigning authority`.